components/tools/OmeroPy/test/integration/test_rawpixelsstore.py

```diff
@@ -8,17 +8,20 @@
    Use is subject to license terms supplied in LICENSE.txt
 
 """
+from __future__ import division
 
+from builtins import hex
```

Comment:
I think this broke https://merge-ci.openmicroscopy.org/jenkins/job/OMERO-test-integration/205/testReport/junit/OmeroPy.test.integration.test_rawpixelsstore/TestRPS/testBigPlane/

I'll try to fix it in https://github.com/ome/openmicroscopy/pull/6127